src/index.js

```diff
@@ -122,6 +123,16 @@ class PeerId {
     return this._idB58String
   }
 
+  // return self-describing String representation
+  // in default format from RFC 0001: https://github.com/libp2p/specs/pull/209
+  toString () {
+    if (!this._idCIDString) {
+      const cid = new CID(1, 'libp2p-key', this.id, 'base32')
+      this._idCIDString = cid.toBaseEncodedString('base32')
```

Comment:
If you cache the CID instance you could use it in a `toCID` also. Note that CID's cache their string representation so calling `toString` on it won't re-encode.

Reply:
As noted above, we don't want to expose CID types in outputs, just Strings.